<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>手动输入</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
      <item>
       <widget class="QLabel" name="item_label">
        <property name="text">
         <string>名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="label_list_box"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="price_per_unit_label">
      <property name="text">
       <string>单价</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="weight_label">
        <property name="text">
         <string>重量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weight_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="price_label">
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="accept_btn">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
